server/activityItem/queries.ts

```diff
@@ -583,13 +583,14 @@ export const createSubmissionUpdatedActivityItem = async (
 		communityId: submission.pub.communityId,
 		collectionId: submission.submissionWorkflow.collectionId,
 		pubId: submission.pub.id,
-		kind: 'submission-updated' as const,
+		kind: 'submission-status-updated' as const,
```

Comment:
This function still needs to be changed so that it only calls `createActivityItem` when the submission status changes (i.e. when `oldSubmission.status !== newSubmission.status`).